rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -421,6 +424,21 @@ public void registerBtcTransaction(
         }
     }
 
+    private void processSvpFundTransactionHashSigned(BtcTransaction transaction) {
+        Optional<Sha256Hash> fundTransactionHashUnsignedOpt = provider.getSvpFundTxHashUnsigned();
+        if (!fundTransactionHashUnsignedOpt.isPresent()) {
+            return;
+        }
+        Sha256Hash fundTransactionHashUnsigned = fundTransactionHashUnsignedOpt.get();
```

Comment:
a bit less verbose could look something like this:

```suggestion
     Sha256Hash fundTransactionHashUnsigned = provider.getSvpFundTxHashUnsigned().orElse(null);
     if (fundTransactionHashUnsigned == null) {
        return;
     }
```
also you might consider using the `ifPresent` method which would look like:
```suggestion
        provider.getSvpFundTxHashUnsigned().ifPresent(fundTransactionHashUnsigned -> { ... });
```

Reply:
mmm, i get your point, but im not sure jaja. I like it better the way it is now

Reply:
i refactored it, take a look. Thanks :)